edgecompute/examples/traffic-routing/url-shortner/edgekv_tokens.js

```diff
@@ -0,0 +1,7 @@
+var edgekv_access_tokens = {
+        "namespace-default": {

```

Comment:
Recommend either not providing a generic edgekv_tokens.js or instead renaming this to "namespace-url-shortener" to be consistent with the namespace used in main.js.